Resources/views/layout.html.twig

```diff
@@ -20,7 +20,7 @@ file that was distributed with this source code.
             {% block sonata_page_breadcrumb %}
                 <div class="row">
                     {% if sonata_seo_context is not defined %}
-                        {% set sonata_seo_context = 'homepage' %}
+                        {% set sonata_seo_context = 'page' %}
```

Comment:
You could have an argument without type hinting. Or maybe, two setters (which would be a BC solution, by the way).